rewrite-java-11/src/main/java/org/openrewrite/java/isolated/ReloadableJava11ParserVisitor.java

```diff
@@ -458,6 +458,11 @@ public J visitClass(ClassTree node, Space fmt) {
                     ),
                     EMPTY
             );
+        } else if (kind.getType() == J.ClassDeclaration.Kind.Type.Enum) {
+            if (positionOfNext(";", null) >= 0) {
+                Space prefix = sourceBefore(";");
+                enumSet = padRight(new J.EnumValueSet(randomId(), prefix, Markers.EMPTY, emptyList(), true), EMPTY);
```

Comment:
Personally I would inline this:
```suggestion
                enumSet = padRight(new J.EnumValueSet(randomId(), sourceBefore(";"), Markers.EMPTY, emptyList(), true), EMPTY);
```

But I'll leave that up to you.